configuration/peloton.conf

```diff
@@ -0,0 +1,119 @@
+# -----------------------------
+# Peloton configuration file
+# -----------------------------
+#
+# This file consists of lines of the form:
+#
+# --name=value
+#
+# Comments are introduced with "#" anywhere on a line.  
+# No whitespace allowed between name & =, and between = & value.
+# The complete list of parameter names and allowed values can be found 
+# in the Peloton documentation.
+#
+# The commented-out settings shown in this file represent the default values.
+# Re-commenting a setting is NOT sufficient to revert it to the default value;
+# you need to reload the server.
+#
+# This file is read on server startup. All parameters, which are marked below, 
+# require a server shutdown and restart to take effect.
+#
+# Any parameter can also be given as a command-line option to the server, e.g.,
+# "peloton -port=15271".  
+# Some parameters can be changed at run time with the "SET" SQL command.
+#
+# Memory units:  kB = kilobytes        Time units:  us  = microseconds
+#                MB = megabytes                     ms  = milliseconds
+#                GB = gigabytes                     s   = seconds
+#                                                   min = minutes
+#                                                   h   = hours
+
+#------------------------------------------------------------------------------
+# FILE LOCATIONS
+#------------------------------------------------------------------------------
+
+# Location of data directory
+--data_directory='data' 
```

Comment:
For parameter like this, how the system can accept it as a command-line argument? I was using GFLAG to parser and accept arguments like "stats_mode" or "port" from command line. But I didn't see the GFLAG declaration of "data_directory".